### ISSUE TITLE
LOGIN element - Forgot Password redirect error

### ISSUE THREAD

**Issue by RonR-WebDesign** [elementor/elementor#9211]
#8774  Prerequisites

- [x ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x ] The issue still exists against the latest stable version of Elementor.

## Description

When doing the "Lost your Password?" action, it accepts an email but then redirects to the above Redirect After Login link.  In our case that it a role protected page and shows an error.

## Steps to reproduce

Edit Login > Additional Options > Redirect After Login > ON

Lost your Password > Show

When doing the "Lost your Password?" action, it accepts an email but then redirects to the above Redirect After Login link.  In our case that it a role protected page and shows an error.

## Isolating the problem

- [x ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here 
[system-info-stsroundtable.com-27-09-2019.txt](https://github.com/elementor/elementor/files/3663559/system-info-stsroundtable.com-27-09-2019.txt)
or in http://pastebin.com/ -->
```
</details>

**Comment by DorShahar** [comment]
Merged issue

I merged this issue with  #8410 , which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.
Thanks for your feedback!

**Comment by d12ilham** [comment]
I have created this plugin for a specific requirement when you are using elementor login widget, you can get it and change it according to your requirements. if anyone wants a quick solution for this issue you can get it from [here](https://github.com/d12ilham/Custom-AJAX-Login-for-Elementor-Popup)
